doc/man7/bio.pod

```diff
@@ -47,10 +47,10 @@ in a memory leak.
 Calling BIO_free_all() on a single BIO has the same effect as calling
 BIO_free() on it other than the discarded return value.
 
-Normally the I<type> argument is supplied by a function which returns a
+Often the I<type> argument is supplied to a function which returns a
```

Comment:
Uhmmmmm, no?  The "*type*" argument is the argument to `BIO_new()`, and for example `BIO_f_base64()` supplies such a "*type*", so the change of `by` to `to` is incorrect.

Reply:
I still think that sentence needs to be rewritten, but I guess that will wait for another change. I undid my change here.